docs/usage/policy.rst

```diff
@@ -46,3 +42,196 @@ Those and additional methods rely on a helper construct ``full_write_guard``, wh
 .. todo::
 
     Describe full_write_guard more in detail and how it works.
+
+Implementing a policy
+---------------------
+
+RestrictedPython only provides the raw material for restricted execution.
+To actually enforce any restrictions, you need to supply a policy
+implementation by providing restricted versions of ``print``,
+``getattr``, ``setattr``, ``import``, etc.  These restricted
+implementations are hooked up by providing a set of specially named
+objects in the global dict that you use for execution of code.
+Specifically:
+
+1. ``_print_`` is a callable object that returns a handler for print
+   statements.  This handler must have a ``write()`` method that
+   accepts a single string argument, and must return a string when
+   called. ``RestrictedPython.PrintCollector.PrintCollector`` is a
+   suitable implementation.
+
+2. ``_write_`` is a guard function taking a single argument.  If the
+   object passed to it may be written to, it should be returned,
+   otherwise the guard function should raise an exception.  ``_write_``
+   is typically called on an object before a ``setattr`` operation.
+
+3. ``_getattr_`` and ``_getitem_`` are guard functions, each of which
+   takes two arguments.  The first is the base object to be accessed,
+   while the second is the attribute name or item index that will be
+   read.  The guard function should return the attribute or subitem,
+   or raise an exception.
+   RestrictedPython ships with a default implementation
+   for ``_getattr_`` which prevents the following actions:
+
+   * accessing an attribute those name start with an underscore
+   * accessing the format method of strings as it is considered harmful.
```

Comment:
```suggestion
   * accessing the format method of strings as this is considered harmful.
```
It would be nicer, if there would be a reason or a link to reason, why this is considered harmful.